### PR TITLE
Evaluating GitHub Activity for Contributors

### PR DESCRIPTION
**Pull request by maelle**
* [x] I have read the Content Guidelines.
* [x] I have read the Technical Guidelines.
* [x] I used or followed the R Markdown or Markdown template.
* [x] I have followed the Style Guide.
* [x] I created or updated my author metadata with correct folder name.
* [x] I have added relevant tags after browsing existing tags (including "community" tag).
* [x] I have added the "tech notes" tag if this is a technote.
* [x] I ran `roblog::ro_lint_md()` on index.md (optional).
* [x] I ran `roblog::ro_check_urls()` on index.md (optional).
* [x] I ran a spell-check on index.md.

I think `roblog::ro_lint_md()` complaints are false positives.

### CODE REVIEW

**Review by steffilazerte — COMMENTED** [review]
Hi @maelle,
I'm just doing a quick review as @yabellini will do the 'official' review. 
Nice informative post! My only comment is really a personal preference, so feel free to stick to your preference :wink:: I generally find that the "for instance" comments break up the flow of the sentence, so I've suggested putting them in brackets at the end of the sentences.

**Comment by maelle** [comment]
Thanks so much @steffilazerte! I agree with you so applied all changes. :smiley_cat:

File: content/blog/2022-07-12-evaluating-github-activity-for-contributors/index.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+Head over the page of continuous integration status for the repositories, for instance it might be the Actions tab.
+Look at the CRAN status if the package was ever on CRAN.
+
+### How it helps
+
+Failing checks that are not tackled might indicate the repository is not actively worked on.
+
+## Conclusion
+
+In this post we summarized some sources of information you as potential contributor can use to assess whether and how much a repository is active, to calibrate your expectations and offers of contributions.
```
Comment (yabellini) [inline content/blog/2022-07-12-evaluating-github-activity-for-contributors/index.md:105]:
```suggestion
In this post we summarized some sources of information you, as potential contributor, can use to assess whether and how much a repository is active, to calibrate your expectations and offers of contributions.
```

**Review by yabellini — COMMENTED** [review]
Just a very minor suggestion (and if not applicable, please discard it).  The text reads very well.  I really like the format of the tip and the purpose of doing so.

File: content/blog/2022-07-12-evaluating-github-activity-for-contributors/index.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  - GitHub
+  - contributors
+  - tech notes
+description: How to evaluate the activity of a GitHub repository as an user or potential contributors 
+---
+
+Say you have a bug report or feature request to make to a package. 
+How can you use information on GitHub to manage your expectations (will there be a quick fix) and actions (should you go ahead and fork the repository)?
+In this post, we shall go over sources of information and explain how they can be used.
+In the end, there is no magical recipe, except perhaps graciousness, as software is made by humans. :wink:
```
[the remark this answers is not in the document]
Comment (maelle) [reply content/blog/2022-07-12-evaluating-github-activity-for-contributors/index.md:17]: :heart:

**Comment by maelle** [comment]
Thank you @yabellini!!

**Pull request merged**